build-tests/x86/tumbleweed/test-image-disk-simple/appliance.kiwi

```diff
@@ -56,6 +56,7 @@
         <package name="kernel-default"/>
         <package name="shim"/>
         <package name="timezone"/>
+        <file name="a_custom_file" target="/etc"/>
```

Comment:
This is not sufficient. You need to be able to set the destination filename.

Reply:
you can. See the implementation, it uses rsync which supports all that, also the parent paths would be created. So all of the following would work:

```
<file name="a_custom_file" target="/etc"/>
<file name="a_custom_file" target="/etc/new_name"/>
<file name="a_custom_file" target="/some/new/path/foo"/>
<file name="a_custom_file" target="/some/new/path/"/>
```

If you want to place the file without a name change into a new new directory your path statement has to end with a "/" such that rsync knows this

From my perspective we are good here

Reply:
Hmm, what about mode and ownership?

Reply:
The sync happens with the default settings that are applied to other sync operations, e.g the overlay tree. These default sync options are:

```
return [
            '--archive', '--hard-links', '--xattrs', '--acls',
            '--one-file-system', '--inplace'
        ]
```

selinux labels are covered by the labeling process and are not related by the time of the syncing of files into the root tree

Reply:
Right, but that doesn't work for the nature of why this PR exists: the OBS VCS doesn't store that data, so we can assume it's always wrong.

Reply:
Sorry I can't follow you. If you place a file as obs source file it's permissions, acls, xattrs should be taken as the file is stored and not changed. To be really honest if obs is not able to store the file with its permissions it's not a kiwi job to fix this. In other words if the file has the wrong permissions after it was imported by the kiwi process and it's because the obs storage backend doesn't care for the origin permissions (which I hardly can believe) then a user needs to fix the permissions via script in config.sh.

In addition we can add another attribute for ```<file>``` that allows to provide a perm and mode mask which we apply after copy. But that I would do in a follow up pull request, not here

Reply:
Uhm it seems really true, file permissions of source files are not preserved when stored in obs.

@Conan-Kudo I added ```owner``` and ```permissions``` attribute to the ```<file>``` element